### PR TITLE
add users page to parliament

### LINKED ISSUE #2118: Parliament Users
Parliament should use Arkime users to authenticate users.

Non parliament user:

- [x] view only

Parliament user
- [x] dismiss issues

Parliament admin:
- [x] dismiss issues
- [x] configure parliament
- [x] update parliament settings

Settings:
- [ ] Add http realm

### PR DESCRIPTION
**Pull request by 31453**
fixes #2118 

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: parliament/vueapp/src/components/user.service.js
Comment (awick) [inline parliament/vueapp/src/components/user.service.js:1]: The other apps call this UserService.js :) 
Some days should all the apps share this file?

Missing the file header

File: parliament/vueapp/src/store.js
Comment (awick) [inline parliament/vueapp/src/store.js:1]: missing file header

File: parliament/parliament.js [hunk 1]
```diff
@@ -1509,6 +1510,17 @@ app.put('/parliament/api/settings', [isAdmin, checkCookieToken], Parliament.apiU
 // Update the parliament general settings object to the defaults
 app.put('/parliament/api/settings/restoreDefaults', [isAdmin, checkCookieToken], Parliament.apiRestoreDefaultSettings);
 
+// user endpoints
+app.get('/parliament/api/user', User.apiGetUser);
+app.post('/parliament/api/users', [jsonParser, User.checkRole('usersAdmin'), setCookie], User.apiGetUsers);
+app.post('/parliament/api/users/csv', [jsonParser, User.checkRole('usersAdmin'), setCookie], User.apiGetUsersCSV);
+app.post('/parliament/api/users/min', [jsonParser, checkCookieToken, User.checkAssignableRole], User.apiGetUsersMin);
```
Comment (awick) [inline parliament/parliament.js:1517]: i'm not sure this api is needed except for role assigners?

Comment (31453) [reply parliament/parliament.js:1517]: yep, you're right

File: parliament/vueapp/src/components/user.service.js
Comment (31453) [reply parliament/vueapp/src/components/user.service.js:1]: Yeah.. but Parliament is special. 

They theoretically can share the same userservice. However, all the stores work differently and the api calls are not the same. I suppose I can pass those things into the common userserice though. I'll investigate

**Review by awick — APPROVED** [review]

**Pull request merged**